src/anyp/PortCfg.h

```diff
@@ -76,6 +78,12 @@ class PortCfg : public CodeContext
 
 } // namespace AnyP
 
+inline std::ostream &
+operator <<(std::ostream &os, const AnyP::PortCfg &p)
```

Comment:
Do we have code (other than clientListenerConnectionOpened and mgr:config reproduction) that prints PortCfg (parts) and can benefit from using the new operator? If yes, we should probably adjust it in this PR to clarify/illustrate the (identification for debugging?) intent behind the new operator.

Reply:
> Do we have code (other than clientListenerConnectionOpened and mgr:config reproduction) that prints PortCfg (parts) and can benefit from using the new operator?

No. This operator is only used in clientListenerConnectionOpened(), mgr:config prints just flags.

Reply:
> code reporting SNMP and HTCP listeners

This code cannot benefit from the operator because it has nothing to do with PortCfg. Listening port values in this case are parsed into a Config.Port field (e.g., Config.Port.icp) and then used by a function (say, icpOpenPorts()) to open listening ports. The startup "Accepting..." debug line prints the corresponding Comm::Connection details.